crates/core/src/config/mod.rs

```diff
@@ -91,6 +91,15 @@ pub struct Config {
     pub start_closed_to_tray: bool,
 }
 
+/// Deserialize `catalog::Source`.
+/// If we find a unknown type, we return None rather than panic.
+fn from_catalog_source<'de, D>(deserializer: D) -> Result<Option<catalog::Source>, D::Error>
```

Comment:
I've made a custom deserializer, because if the user had selected Townlong Yak as the `catalog::source` it would fail when loading `Config`. Now, it just returns `None` if it does not know the enum type.

Reply:
You could also alias source like you did for RepoKind and it'll just load that one?

Reply:
I think it's better to keep it the same so I'll do alias as you suggest!